### PR TITLE
grid done

### PR DESCRIPTION
**Pull request by theVivek-kumar**
Grid done 
Rating done
Model
please review

### CODE REVIEW

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go 👍

File: grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+.checked {
+
+    color: var(--ORANGE);
+}
+
+.unchecked {
+    color: var(--GREY);
+}
+.rating--title{
+    color: aliceblue;
```
Comment (dev-ashishsethi) [inline grid/grid.css:76]: use variable for colors

File: grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    padding: 2rem 2.5rem;
+    margin: 2rem 0;
+    width: 50%;
+    margin: auto;
+}
+
+.rating {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 20px;
```
Comment (dev-ashishsethi) [inline grid/grid.css:62]: use rems

**Review by dev-ashishsethi — APPROVED** [review]
do check comments

**Pull request merged**